configs/projects/puppet-agent.rb

```diff
@@ -92,6 +92,7 @@
   proj.component "puppet"
   proj.component "facter"
   proj.component "hiera"
+  proj.component "leatherman"
   proj.component "marionette-collective"
   proj.component "cpp-pcp-client"
   proj.component "pxp-agent"
```

Comment:
Yes, but they're still using their vendored version. Until I get around to unvendoring lth from their, they don't have a require on the system version